src/components/ItemEditorPage/RightPanel/RightPanel.tsx

```diff
@@ -701,7 +699,7 @@ export default class RightPanel extends React.PureComponent<Props, State> {
                         ) : null}
                       </Collapsable>
                     )}
-                    {item && isSmart(item) && (
+                    {item && isSmart(item) && item.data.requiredPermissions?.length && (
```

Comment:
I think this will show a "0" if the item exists, is smart, and has an empty array of permissions. Would you mind refactoring this using a ternary operator?

Reply:
good catch! fixed